kcidev/libs/dashboard.py

```diff
@@ -8,7 +8,7 @@
 
 from kcidev.libs.common import *
 
-DASHBOARD_API = "https://dashboard.kernelci.org/api/"
+DASHBOARD_API = "https://dashboard.kernelci.org:9000/api/"
```

Comment:
Also, the option `interval_in_days` only works when port number is specified.
Could you please explain the reason? @aliceinwire @nuclearcat 
Do we have multiple instances running for dashboard APIs?

Reply:
> So do you mean output differs for `$ kci-dev --settings ./kci-dev.toml results trees --days 1` and `$ kci-dev --settings ./kci-dev.toml results trees --days 7` It doesn't in my case.

I see. That's the same result also on my side.

Reply:
Yes, it only differs with the port number specified.
Need to ask on the dashboard repo.